js/layout-javascript/agenda-code.js

```diff
@@ -606,6 +609,8 @@ DynamicList.prototype.attachObservers = function() {
       var result;
       var id = _this.$container.find('.agenda-detail-wrapper[data-entry-id]').data('entry-id');
 
+      $('.agenda-list-holder').removeClass('hidden');
```

Comment:
```suggestion
      _this.$container.find('.agenda-list-holder').removeClass('hidden');
```

Reply:
@tonytlwu i researched all my pr and find some lines where this code is needed and added it